project/SparkBuild.scala

```diff
@@ -713,85 +756,9 @@ object SparkConnectCommon {
   }
 }
 
-object SparkConnect {
-  import BuildCommons.protoVersion
-
+object SparkConnectServer {
```

Comment:
This is only here to make assembly work. It is a bit weird because this module does not need a full assembly anymore.